irc/halloy/Portfile

```
@@ -36,472 +36,586 @@ destroot {
+    xdg-home                         1.3.0  ec1cdab258fb55c0da61328dc52c8764709b249011b2cad0454c72f0bf10a1f6 \
     xkbcommon-dl                     0.4.2  d039de8032a9a8856a6be89cea3e5d12fdd82306ab7c94d74e6deab2460651c5 \
-    xkeysym                          0.2.0  054a8e68b76250b253f671d1268cb7f1ae089ec35e195b2efb2a4e9a836d0621 \
-    xml-rs                          0.8.19  0fcb9cbac069e033553e8bb871be2fbdffcab578eb25bd0f7c508cedc6dcd75a \
-    xxhash-rust                     0.8.10  927da81e25be1e1a2901d59b81b37dd2efd1fc9c9345a55007f09bf5a2d3ee03 \
+    xkeysym                          0.2.1  b9cc00251562a284751c9973bace760d86c0276c471b4be569fe6b068ee97a56 \
+    xml-rs                          0.8.21  539a77ee7c0de333dcc6da69b177380a0b81e0dacfa4f7344c465a36871ee601 \
     yansi                            1.0.1  cfe53a6657fd280eaa890a3bc59152892ffa3e30101319d168b781ed6529b049 \
     yazi                             0.1.6  c94451ac9513335b5e23d7a8a2b61a7102398b8cca5160829d313e84c9d98be1 \
-    zbus                            3.15.2  675d170b632a6ad49804c8cf2105d7c31eddd3312555cffd4b740e08e97c25e6 \
-    zbus_macros                     3.15.2  7131497b0f887e8061b430c530240063d33bf9455fa34438f388a245da69e0a5 \
-    zbus_names                       2.6.1  437d738d3750bed6ca9b8d423ccc7a8eb284f6b1d6d4e225a0e4e6258d864c8d \
+    zbus                             4.4.0  bb97012beadd29e654708a0fdb4c84bc046f537aecfde2c3ee0a9e4b4d48c725 \
+    zbus_macros                      4.4.0  267db9407081e90bbfa46d841d3cbc60f59c0351838c4bc65199ecd79ab1983e \
+    zbus_names                       3.0.0  4b9b1fef7d021261cc16cba64c351d291b715febe0fa10dc3a443ac5a5022e6c \
     zeno                             0.2.3  dd15f8e0dbb966fd9245e7498c7e9e5055d9e5c8b676b95bd67091cd11a1e697 \
-    zerocopy                        0.7.32  74d4d3961e53fa4c9a25a8637fc2bfaf2595b3d3ae34875568a5cf64787716be \
-    zerocopy-derive                 0.7.32  9ce1b18ccd8e73a9321186f97e46f9f04b778851177567b1975109d26a08d2a6 \
+    zerocopy                        0.7.35  1b9b4fd18abc82b8136838da5d50bae7bdea537c574d8dc1a34ed098d6c166f0 \
+    zerocopy-derive                 0.7.35  fa4f8080344d4671fb4e831a13ad1e68092748387dfc4f55e356242fae12ce3e \
+    zeroize                          1.8.1  ced3678a2879b30306d323f4542626697a464a97c0a07c9aebf7ebca65cd4dde \
     zune-inflate                    0.2.54  73ab332fe2f6680068f3582b16a24f90ad7096d5d39b974d1c0aff0125116f02 \
-    zvariant                        3.15.2  4eef2be88ba09b358d3b58aca6e41cd853631d44787f319a1383ca83424fb2db \
+    zvariant                         4.2.0  2084290ab9a1c471c38fc524945837734fbf124487e105daec2bb57fd48c81fe \
+    zvariant_derive                  4.2.0  73e2ba546bda683a90652bac4a279bc146adad1386f25379cf73200d2002c449 \
+    zvariant_utils                   2.1.0  c51bcff7cc3dbb5055396bcf774748c3dab426b4b8659046963523cee4808340
```

We're missing a line continuation here, causing an error parsing this Portfile:

```suggestion
zvariant_utils                   2.1.0  c51bcff7cc3dbb5055396bcf774748c3dab426b4b8659046963523cee4808340 \
```

Can you fix by amending the current commit?

---

done.